### PR TITLE
release 1.26.0

### PR DESCRIPTION
**Pull request by cqc-melf**
# Description

Please summarise the changes.

# Related issues

Please mention any github issues addressed by this PR.

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

File: examples/example_requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
-pytket-cirq == 0.34.0
-pytket-qiskit == 0.48.0
-pytket-qujax == 0.17.0
+pytket == 1.26.0
+pytket-cirq == 0.35.0
+pytket-qiskit == 0.50.0
+pytket-qujax == 0.18.0
 openfermion 
 pytest
+pydantic == 2.6.4
```
Comment (CalMacCQ) [inline examples/example_requirements.txt:7]: Why is this specific version of pydantic pinned?

File: examples/ci-tested-notebooks.txt [hunk 2]
```diff
@@ -6,7 +6,6 @@ compilation_example
 conditional_gate_example
 contextual_optimization
 creating_backends
-Forest_portability_example
```
Comment (CalMacCQ) [inline examples/ci-tested-notebooks.txt:9]: Should we also move the notebook itself? Or were you planning to do that in a separate PR?

**Review by CalMacCQ — COMMENTED** [review]
Looks good, just a few small clarifications and then I'm happy to approve.

File: manual/manual_assertion.rst [hunk 3]
```diff
@@ -78,7 +78,7 @@ If there is noise in the device, which is the case for all devices in the NISQ e
 
 .. jupyter-input::
 
-    from qiskit.providers.aer.noise import NoiseModel
+    from qiskit_aer.noise import NoiseModel
     from qiskit import IBMQ
     IBMQ.load_account()
```
[the remark this answers is not in the document]
Comment (cqc-melf) [reply manual/manual_assertion.rst:83]: I think we should do this separately. There are quiet a few things that are deprecated, I have only updated what stopped working.

File: examples/ci-tested-notebooks.txt (hunk 2 above)
Comment (cqc-melf) [reply examples/ci-tested-notebooks.txt:9]: You mean to the legacy directory? I would have hoped that we would be able to add it back in soon and can just keep it here?

File: examples/example_requirements.txt (hunk 1 above)
Comment (cqc-melf) [reply examples/example_requirements.txt]: That was only for testing, remove it again in [33e069a](https://github.com/CQCL/pytket-docs/pull/310/commits/33e069a2a1e3300f9af430c4709e28eece005771)

Thank you for spotting!

**Review by CalMacCQ — APPROVED** [review]
